package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "example_module_mailtheme",
+  "version": "1.0.0",
+  "description": "Example module for PrestaShop mail theme",
+  "main": "install.js",
+  "scripts": {
+    "build": "webpack --progress --colors --debug --display-chunks --mode=production",
+    "dev": "webpack --progress --colors --debug --display-chunks --watch --mode=development",
+    "watch": "npm run dev",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/PrestaShop/example_module_mailtheme.git"
+  },
+  "keywords": [
+    "PrestaShop",
+    "email",
+    "example"
+  ],
+  "author": "PrestaShop SA <contact@prestashop.com>",
+  "license": "OSL-3.0",
```

Comment:
AFL for modules? :thinking:

Reply:
since it's an example module I thought OSL was right, not sure though